scripts/local/scripts/environment/scan-ssh.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# Copyright 2022-2023 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+THIS_DIR=$(cd "$(dirname "$0")" && pwd)
+ENVIRONMENTS_DIR=$(cd "$THIS_DIR" && cd ../../environments && pwd)
+
+if [ "$#" -gt 0 ]; then
+	INSTANCE_ID=$1
+	echo "$INSTANCE_ID" > "$ENVIRONMENTS_DIR/current.txt"
```

Comment:
The indentation of these lines looks wrong.

Reply:
Sorted, thanks